gemma-core/src/main/java/ubic/gemma/persistence/service/analysis/expression/diff/DifferentialExpressionAnalysisDaoImpl.java

```diff
@@ -338,13 +338,8 @@ public void thaw( DifferentialExpressionAnalysis differentialExpressionAnalysis
         StopWatch timer = new StopWatch();
         timer.start();
 
-        Session session = this.getSessionFactory().getCurrentSession();
-        session.clear();
-
-        session.buildLockRequest( LockOptions.NONE ).lock( differentialExpressionAnalysis );
```

Comment:
I think this becomes totally unnecessary with DEA service being now transactional.